coreos-assembler

```diff
@@ -12,6 +12,16 @@ case $USER in
     *) echo "Executed as non-builder user; assuming sudo rights..." 1>&2;;
 esac
 
+# Ensure we've unshared our mount namespace so
+# the later umount doesn't affect the host potentially
+if [ -z "${coreos_assembler_unshared:-}" ]; then
+    exec sudo -- env coreos_assembler_unshared=1 unshare -m -- runuser -u ${USER} -- $0 "$@"
```

Comment:
starting to get deep :) `exec runuser`,  `exec unshare -- runuser`  :) - computers are amazing

Reply:
Yeah I had to think 3-4 times about that line :smile: